### ISSUE TITLE
Support for R1 and WA keys with a new verification method type

### ISSUE THREAD

**Issue by ghost** [Gimly-Blockchain/eosio-did-spec#13]
Currently the EOSIO DID does not officially support the R1 and WA key types. This requires creating a new verification method type as follows:
1. Fork the repository of the [EcdsaSecp256k1Signature2019](https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019)
2. Make appropriate changes to the repository to adapt it for the [Secp256r1](https://neuromancer.sk/std/secg/secp256r1) curve
3. Create a proposal to the W3C Credentials Community Group. See [instructions](https://docs.google.com/document/d/1vj811aUbs8GwZUNo-LIFBHafsz4rZTSnRtPv7RQaqNc)
4. Once approved and finished, create a PR to the [DID Specification Registries](https://w3c.github.io/did-spec-registries/#verification-method-types) with a link to the new type

tagging @OR13 as you may be interested in this issue

**Comment by ghost** [comment]
See Issue #5 for background

**Comment by theblockstalk** [comment]
Just realized that the Secp256r1 curve can actually be represented using the [JsonWebKey2020](https://w3c.github.io/did-spec-registries/#jsonwebkey2020) verification method type with a Jwk:
```js
{
  crv: "P-256"
  "x": "NtngWpJUr-rlNNbs0u-Aa8e16OwSJu6UiFf0Rdo1oJ4",
  "y": "qN1jKupJlFsPFc1UkWinqljv4YE0mq_Ickwnjgasvmo",
  "kty": "EC"
}
```

We should update the spec to say this should be used

**Comment by theblockstalk** [comment]
this was merged in PR #19
